src/main/java/jenkins/plugins/http_request/HttpRequestGlobalConfig.java

```diff
@@ -28,7 +28,11 @@
 @Extension
 public class HttpRequestGlobalConfig extends GlobalConfiguration {
 
-    private List<BasicDigestAuthentication> basicDigestAuthentications = new ArrayList<>();
+    /**
+     * @deprecated removed without replacement
+     */
+    @Deprecated
+    private transient List<BasicDigestAuthentication> basicDigestAuthentications = new ArrayList<>();
```

Comment:
Usage of deprecated field still present in `HttpRequestGlobalConfig#getAuthentications`

Reply:
I have removed the usage from `getAuthentications()`. The other two usages are getter and setter.

What is the best scenario for those? If getter / setter are deprecated but still available I'd suggest keeping the related test cases too (it's still some kind of public API), or remove getter, setter field and related tests altogether.

Reply:
Please remove all callers of deprecated methods but retain the methods for binary compatibility